sklearn/model_selection/_search.py

```diff
@@ -1127,6 +1127,13 @@ class RandomizedSearchCV(BaseSearchCV):
     It is highly recommended to use continuous distributions for continuous
     parameters.
 
+    Note that before SciPy 0.16, the ``scipy.stats.distributions`` do not
+    accept a custom RNG instance and always use the singleton RNG from
+    ``numpy.random``. Hence setting ``random_state`` will not guarantee a
+    deterministic iteration whenever ``scipy.stats`` distributions are used to
+    define the parameter search space. Deterministic behavior is however
+    guaranteed from SciPy 0.16 onwards.
```

Comment:
It's guaranteed for scipy >= 0.16 only if the random seed is provided. 

I would remove this last sentence, as it's a bit redundant.

Reply:
Please fix this

Reply:
Fixed.